cmd/context/create.go

```diff
@@ -201,7 +201,7 @@ func (c *ContextCommand) UseContext(ctx context.Context, ctxOptions *ContextOpti
 	}
 
 	if ctxOptions.IsCtxCommand {
-		oktetoLog.Success("Using context %s @ %s", okteto.Context().Namespace, okteto.RemoveSchema(ctxStore.CurrentContext))
+		oktetoLog.Success("Using %s @ %s", okteto.Context().Namespace, okteto.RemoveSchema(ctxStore.CurrentContext))
```

Comment:
I think this string is used in more places. Can you create a constant and use it in every place we use this string message?

Reply:
I know there is this usage here: https://github.com/okteto/okteto/blob/ee65cf263db00e84bc347f8095742b614641577a/cmd/context/use.go#L137, which looks quite similar but not identical. Maybe we could merge the two into one `if isCtxCommand || show` check? Would have to find other uses of `UseContext` though. Not sure.

Reply:
Let's keep them with different strings for now since one is used for changing the namespace and the other one when changing context.